DemoApp/Sources/Components/SnapshotTrigger.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  SnapshotTrigger.swift
```

Comment:
Stream header.

Reply:
Surprising why swift-format didn't catch it 🤔

Reply:
Found it #285